apps/web/src/services/my-page/my-account/user-account/modules/NotificationEmail.vue

```diff
@@ -100,15 +91,23 @@ const {
     invalidState,
     invalidTexts,
 } = useFormValidator({
-    notificationEmail: state.email,
+    notificationEmail: '',
 }, {
     notificationEmail(value: string) { return !emailValidator(value) ? '' : i18n.t('IDENTITY.USER.FORM.EMAIL_INVALID'); },
 });
 
 /* Watcher */
 watch(() => store.state.user.email, (value) => {
-    setForm('notificationEmail', value);
-});
+    let result = value;
+    if (value === '') {
```

Comment:
nitpick: Is there a possibility for the `value` to be `undefined`? (Or else, ignore this comment.)
If so, how about making the following changes?

```suggestion
    if (value) {
```